src/main/java/ch/wisv/chue/hue/HueLightState.java

```diff
@@ -0,0 +1,108 @@
+package ch.wisv.chue.hue;
+
+import com.philips.lighting.hue.sdk.utilities.PHUtilities;
+import com.philips.lighting.model.PHLight;
+import com.philips.lighting.model.PHLightState;
+import javafx.scene.paint.Color;
+
+import java.util.Optional;
+import java.util.OptionalInt;
+
+public class HueLightState {
+    private Optional<AlertMode> alertMode = Optional.empty();
+    private Optional<EffectMode> effectMode = Optional.empty();
+    private OptionalInt transitionTime = OptionalInt.empty();
+    private Optional<Color> color = Optional.empty();
+
+    protected static PHLightState asPHLightState(HueLightState hueLightState) {
```

Comment:
This class shouldn't depend on `com.philips.lighting.*`, this method can be moved to `PhilipsHueFacade`.